common/i18n/resources/en.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     srs: 'SRS',
     errorMessage: 'Error message',
     errorMessage_plural: 'Error messages',
+    invalid: 'INVALID',
```

Comment:
should the uppercase be managed via css?

Reply:
Yes, changing to lowercase

Reply:
yes. let's also mange it via css